docs/api-guide/fields.md

```diff
@@ -366,6 +366,7 @@ Used by `ModelSerializer` to automatically generate fields if the corresponding
 - `allow_blank` - If set to `True` then the empty string should be considered a valid value. If set to `False` then the empty string is considered invalid and will raise a validation error. Defaults to `False`.
 - `html_cutoff` - If set this will be the maximum number of choices that will be displayed by a HTML select drop down. Can be used to ensure that automatically generated ChoiceFields with very large possible selections do not prevent a template from rendering. Defaults to `None`.
 - `html_cutoff_text` - If set this will display a textual indicator if the maximum number of items have been cutoff in an HTML select drop down. Defaults to `"More than {count} items…"`
+- `use_value` - If set to `True` then input and output values changed with choice field values.
```

Comment:
I believe that the option should be something more akin to `use_human_readable`. This is a more explicit kwarg name. Perhaps a more suitable doc entry would be

- `use_human_readable` - If set to `True` then use the human-readable values from the choice option tuple for input/output. For example, with

```python
choices=(
    (1, 'seller'),
    (2, 'buyer'),
)
```

`seller` and `buyer` are used for input and output.